app/controllers/ContactEmailAddressController.scala

```diff
@@ -62,7 +62,7 @@ class ContactEmailAddressController @Inject()(
           val (pageTitle, heading) = request.userAnswers.get(ContactNamePage) match {
             case Some(name) =>
               (Messages("contactEmailAddress.business.title"),
-                Messages("contactEmailAddress.business.heading", s"${name.firstName} ${name.secondName}"))
+                Messages("contactEmailAddress.business.heading", s"$name"))
```

Comment:
if name is a string you can pass it in itself, here you're interpolating to a string unnecessarily. I'm not going to tag but it's the same with all the others